### DISCUSSION TITLE
[Feature] Filter by time period

### DISCUSSION THREAD

**Discussion by johndoe0815** [damongolding/immich-kiosk#156] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Besides selecting albums or people, the main selection criteria I look for is time/years.
My (external) library contains pictures of the last 15 years. Sometimes it is nice to have a slide show over everything, but often I rather want one of e.g. just the current year. So that would require a parameter that lets you select a certain year, or from and to dates, whatever is easier.

**Comment by damongolding** [comment]
Added in 0.15.0
